### PR TITLE
[SPARK-28541][WEBUI] Document Storage page

### PR DESCRIPTION
**Pull request by zhengruifeng**
## What changes were proposed in this pull request?
add an example for storage tab

## How was this patch tested?
locally building

### CODE REVIEW

**Comment by zhengruifeng** [comment]
图片

File: docs/web-ui.md [hunk 1]
```diff
@@ -45,6 +45,53 @@ The Storage tab displays the persisted RDDs and DataFrames, if any, in the appli
...
+
+<p style="text-align: center;">
+  <img src="img/webui-storage-detail.png"
+       title="Storage detail"
+       alt="Storage detail"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+Cliking the RDD name 'rdd' displays the details of data persistance, such as the data distribution on the cluster.
```
Comment (xuanyuanking) [inline docs/web-ui.md]: nit: could you check the display of 'rdd'?
image

**Review by xuanyuanking — COMMENTED** [review]
LGTM
Thanks for this work! cc @gatorsmile

File: docs/web-ui.md (hunk 1 above)
Comment (zhengruifeng) [reply docs/web-ui.md]: Sorry, I do not get the point. Is there a rendering issue?
In the example, I set the name of the first RDD to `rdd`.
Thanks for reviewing!

**Comment by zhengruifeng** [comment]
图片

File: docs/web-ui.md (hunk 1 above)
Comment (gatorsmile) [reply docs/web-ui.md]: How about "You can click the RDD name 'rdd' for obtaining the details .... " ?

**Review by gatorsmile — COMMENTED** [review]
Thanks for adding the storage page.

File: docs/web-ui.md [hunk 2]
```diff
@@ -45,6 +45,54 @@ The Storage tab displays the persisted RDDs and DataFrames, if any, in the appli
...
+  <img src="img/webui-storage-tab.png"
+       title="Storage tab"
+       alt="Storage tab"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+After running the above example, we can find two RDDs listed in the Storage tab. Basic information like
+storage level, number of partitions and memory overhead are provided. Note that the newly persisted RDDs
+or DataFrames are not shown in the tab before they are materialized, to monitor a specific RDD or DataFrame,
```
Comment (gatorsmile) [inline docs/web-ui.md]: "materialized, to" -> "materialized. To

File: docs/web-ui.md [hunk 3]
```diff
@@ -45,6 +45,54 @@ The Storage tab displays the persisted RDDs and DataFrames, if any, in the appli
...
+
+<p style="text-align: center;">
+  <img src="img/webui-storage-detail.png"
+       title="Storage detail"
+       alt="Storage detail"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+You can click the RDD name 'rdd' for obtaining the details of data persistance, such as the data
```
Comment (srowen) [inline docs/web-ui.md]: persistance -> persistence

**Review by srowen — CHANGES_REQUESTED** [review]
Similar question about whether we want a screenshot with a Spark version number in it. The header could just be removed. I don't think it's a big deal either way.

**Comment by zhengruifeng** [comment]
@srowen Thanks for reviewing! I had move the header outside in https://github.com/apache/spark/pull/25445/commits/d4d1ca43bcb6f49301d82a68d51e3844a71a8576

There were two screenshots of the built docs, and I just deleted the old one with version number in it.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**